system/jlib/jsecrets.cpp

```diff
@@ -410,12 +421,24 @@ class CVault
 
         std::string json;
         json.append("{\"role_id\": \"").append(appRoleId).append("\", \"secret_id\": \"").append(appRoleSecretId).append("\"}");
+
+        PROGLOG("preparing to post approle login to %s", schemeHostPort.str());
```

Comment:
Is this going to cause excessive logging once everything is working?

Reply:
@ghalliday when working, approle login would only happen initially, and then once the returned token next expires.  But even so, I can further reduce the PROGLOGs, especially since I now log more errors inside httplib.

Reply:
@ghalliday I've reduced the excessive logging.  Removing some, and moving some to the vault constructor where it will only happen once.